src/shared/ui/modal-window/modal-window.vue

```diff
@@ -0,0 +1,17 @@
+<script setup lang="ts">
+import { modal, modalClose } from '@/shared/lib/modal'
+</script>
+
+<template>
+  <dialog class="modal" open aria-live="assertive" @keydown.esc="modalClose()">
+    <header class="modal__header">
+      <h3 id="modal-title" class="modal__title">{{ modal.name }}</h3>
+      <button class="modal__button"  aria-labelledby="close modal-title" @click="modalClose()">
+        X
+      </button>
+    </header>
+    <article class="modal__article">
```

Comment:
Многострочные элементы одного уровня должны быть отделены друг от друга отступом.

```suggestion
    </header>

    <article class="modal__article">
```